common/app/model/dotcomrendering/DotcomOnwardsRenderingDataModel.scala

```diff
@@ -0,0 +1,37 @@
+package model.dotcomrendering
+
+import model.ContentFormat
+import model.dotcomrendering.pageElements.PageElement
+import play.api.libs.json.{Json, Writes}
+
+// -----------------------------------------------------------------
+// DCR Onwards DataModel
+// -----------------------------------------------------------------
+
+case class DotcomOnwardsRenderingDataModel(
+    heading: String,
+    trails: Seq[OnwardItem],
+    ophanComponentName: String,
+    isCuratedContent: Boolean,
+)
```

Comment:
We need to add `format` here. The latest DCR version of this model now expects

```ts
type CAPIOnwardsType = {
	heading: string;
	trails: CAPITrailType[];
	ophanComponentName: OphanComponentName;
	format: CAPIFormat;
	isCuratedContent?: boolean;
};
```

where

```ts
type CAPIFormat = {
	design: CAPIDesign; // string
	theme: CAPITheme; // string
	display: CAPIDisplay; // string
};
```

Reply:
Format is available on each "trail" (CAPITrailType). If we do want to add format at this level, what format would make sense on a collection of trails?

Reply:
Hmm, this is more complex than I thought. We need to know the format to decide colouring for the container title and dot
<img width="449" alt="Screenshot 2022-07-27 at 15 23 56" src="https://user-images.githubusercontent.com/1336821/181272061-3aab4d26-5a7b-4215-8f3c-f745c0182620.png">


I initially thought we'd have the article id in the request. Most onwards calls do. 

Related example
https://code.api.nextgen.guardianapps.co.uk/related/business/2022/jul/27/china-property-sales-could-plunge-by-one-third-analysts-say-as-crisis-deepens.json?dcr=true

But for curated content that isn't the case so there's no way to know which format to use 🤔 

Curated example
https://code.api.nextgen.guardianapps.co.uk/container/data/754c-8e8c-fad9-a927.json?html